lang/python27/Portfile

```diff
@@ -64,40 +63,65 @@ depends_lib         port:bzip2 \
 
-openssl.branch      1.1
+
+if {${subport} eq ${name}} {
+    PortGroup       openssl 1.0
+
+    openssl.branch  1.1
+}
 
 # This port is used by clang-3.4 to bootstrap libcxx
 subport ${name}-bootstrap {
-    set stdprefix ${prefix}
-    prefix ${prefix}/libexec/libcxx-bootstrap
+    revision                [ expr ${revision} + 1 ]
+
+    use_xz                  no
+    depends_extract         port:xz-bootstrap
+    depends_skip_archcheck-append \
+                            xz-bootstrap
+    extract.suffix          .tar.xz
+    extract.cmd             ${prefix}/libexec/xz-bootstrap/bin/xz
+
+    prefix                  ${prefix}/libexec/${subport}
     set frameworks_dir      ${prefix}/Library/Frameworks
     set applications_dir    ${prefix}/Applications
-    configure.cppflags-append   -I${stdprefix}/include
-    configure.ldflags-append    -L${stdprefix}/lib
-    configure.env-append    PATH=${prefix}/bin:$::env(PATH)
+
+    patchfiles-delete       patch-libedit.diff
+
+    # Avoid macports-clang dep (doesn't use C++ anyway)
     configure.cxx_stdlib
     compiler.whitelist      clang llvm-gcc-4.2 gcc-4.2 apple-gcc-4.2
-    patchfiles-delete       patch-libedit.diff
-    depends_build-replace   port:pkgconfig port:pkgconfig-bootstrap
-    depends_lib-replace     port:ncurses port:ncurses-bootstrap
-    depends_lib-delete      port:db48 \
-                            port:gettext-runtime \
-                            port:libedit \
-                            port:sqlite3
-    # no need to be 'port select'able
+
+    # sterilize MacPorts build environment; we want nothing picked up from MP prefix
+    compiler.cpath
+    compiler.library_path
+
+    configure.cflags
+    configure.cxxflags
+    configure.cppflags
+    configure.optflags
+    configure.ldflags
+
+    configure.universal_cflags
+    configure.universal_cxxflags
+    configure.universal_cppflags
+    configure.universal_ldflags
+    configure.universal_args
+
+    configure.ccache    no
+    configure.distcc    no
+
+    # sterilize PATH
+    configure.env-append    PATH=/usr/bin:/bin:/usr/sbin:/sbin
+    build.env-append        PATH=/usr/bin:/bin:/usr/sbin:/sbin
+
+    # sterilize dependencies
+    depends_build
+    depends_lib
     depends_run
-    use_xz              no
-    depends_extract     port:xz-bootstrap
-    depends_skip_archcheck-append   xz-bootstrap
-    extract.suffix      .tar.xz
-    extract.cmd         ${prefix}/bin/xz
-    if {${os.platform} eq "darwin" && ${os.major} >= 18} {
-        # libstdc++ can't be used on Mojave and later
-        known_fail          yes
-    }
-    configure.args      ac_cv_header_libintl_h=no \
-                        ac_cv_lib_intl_textdomain=no \
-                        ac_cv_search_bind_textdomain_codeset=no
+
+    configure.args          ac_cv_header_libintl_h=no \
+                            ac_cv_lib_intl_textdomain=no \
+                            ac_cv_search_bind_textdomain_codeset=no
```

Comment:
Is this still necessary and appropriate? Shouldn't using the system gettext be OK?

Reply:
@jmroot it was here before, and I can try to remove it, but it may broke the port somewhere. I prefer to keep it here. Do you suggest to remove it?

Reply:
If you're worried about changes breaking the port, you probably shouldn't make all these other changes. :)

This was only here to prevent gettext from being found in `$prefix` (b68b114), so it should no longer be needed with the flags cleared of references to `$prefix`.